fendermint/app/config/default.toml

```diff
@@ -40,7 +40,7 @@ read_request_concurrency = 10
 blob_metrics_interval = 10
 
 # Gas limit used by the system actor to manage blob queues.
-blob_queue_gas_limit = 500000000
+blob_queue_gas_limit = 1000000000
```

Comment:
more conservative... this still allows 10 of this tx type in a single block, which is fine for now. the risk is too high to not be very conservative here